R/conjunct_test_linter.R

```diff
@@ -32,6 +33,12 @@
 #'   linters = conjunct_test_linter(allow_named_stopifnot = FALSE)
 #' )
 #'
+#' lint(
+#'   text = "dplyr::filter(mtcars,
+#'                         mpg > 20 & vs == 0)",
```

Comment:
please use tidyverse style in examples

Reply:
Ok, see https://github.com/r-lib/lintr/pull/2110/commits/f5c599b19a9b03d6d6ed3e71f53dc95894b76069